site/app/templates/polls/PollForm.twig

```diff
@@ -140,20 +140,33 @@
             <option value="when_ended" {{ poll is not null and poll.getReleaseAnswer() == 'when_ended' ? "selected" : "" }}>After Poll Ended</option>
             <option value="always" {{ poll is not null and poll.getReleaseAnswer() == 'always' ? "selected" : "" }}>Always</option>
         </select>
-        <br/>
+        <br><br>
+        <div>
+            <label for = "poll-custom-options" class="option-title credit-form-item">
+                Allow student-written responses?
+            </label>
+            <input id ="poll-custom-options" class="credit-box credit-form-item" type="checkbox" name="poll-custom-options" {{poll is not null and poll.getAllowsCustomResponses() ? "checked" : ""}}>
+            <i>
+                <br />
+                Are you certain you want to allow student-written responses? Students will be able to submit responses that will be visible to other students.<br />
+                Responses may become cluttered. If the question-type is not a survey, these responses will be labeled incorrect by default.
+            </i>
+        </div>
 
         <hr>
         <h2> Responses: </h2>
+        <div {{is_survey ? "style='display:none'" : "style='display:inline-block'"}}> <!-- Show/Hide 'Toggle All' based on whether the poll is a survey -->
+            <input aria-label="Toggle All" id = "toggle-all" type="checkbox" name="toggle-all-options" onchange = "togglePollFormOptions()">
```

Comment:
Similar whitespace issues appear throughout the diff in this file.

```suggestion
            <input aria-label="Toggle All" id="toggle-all" type="checkbox" name="toggle-all-options" onchange="togglePollFormOptions()">
```

Reply:
[424c581](https://github.com/Submitty/Submitty/pull/10197/commits/424c5815130a73f00f64cf3e35e525562c1f0058)